src/main/flight/gps_rescue.c

```diff
@@ -272,7 +272,7 @@ static void rescueAttainPosition()
     case RESCUE_DO_NOTHING:
         gpsRescueAngle[AI_PITCH] = 0.0f;
         gpsRescueAngle[AI_ROLL] = 0.0f;
-        rescueThrottle = gpsRescueConfig()->throttleHover;
+        rescueThrottle = gpsRescueConfig()->throttleHover - 50;
```

Comment:
Not sure here. We should only descent if `RESCUE_DESCENT` or `RESCUE_LANDING` were triggered before.

Reply:
'Do nothing' is a kind of 'temporary holding pattern', where previously we had an immediate disarm.
It gets activated in place of the disarm in case the user was using switch failsafe but encounters a sanity check that would otherwise disarm.
Problem is that 'floating around doing nothing' doesn't seem to encourage people to give up, and they just get a bit confused.
And I think in general it should start to descend, slowly.
The main problem is that if hover throttle is a little high, it will climb.
Open to suggestions here but feel that this is probably the right thing to do.  If up high it is of no consequence, if low it is a kind of simple landing procedure that does not rely on GPS.

Reply:
I see what you try to solve here but logical it should not do this in the RESCUE_DO_NOTHING state. Instead in this case it should go back to DESCENT or LANDING phase or use the steps for the decrease. We also could make it a config correction value or even another case.

Reply:
Renaming `RESCUE_DO_NOTHING` to `RESCUE_CONTINUE` would make sense to me,

Reply:
Though 'do nothing' comes into force only when there is no GPS signal, or the rescue otherwise cannot continue to do what it should, and when the rescue was initiated with sticks.  It is specifically intended to kind of just 'do nothing', since it doesn't know what to do anymore.  After 10s of this, it will disarm and crash.  
On reflection, it makes more sense to start gradually descending than holding the altitude, or worse, climbing.
The best solution would be to also show an error message saying 'GPS Rescue Failure!', but we don't have that option at present.